drivers/SmartThings/matter-thermostat/src/init.lua

```diff
@@ -374,11 +405,15 @@ local function set_setpoint(setpoint)
   end
 end
 
-local function setpoint_limit_handler(limit_field)
+local function setpoint_limit_handler(limit_field, limit_field_opposite, attribute)
```

Comment:
You could modify this to follow the same strategy of indicating `min/max` in the parameters as you have done above for the `temp_attr_handler_factory`

Reply:
Updated as suggested in c33ee3ad996ac550630fc72a2fd778f617096866.